python-oop-si-exercises-martikecskemeti/3-phone-numbers/person.py

```diff
@@ -3,18 +3,20 @@ class Person():
     _phone_number = None
 
     def __init__(self, name, phone_number):
-        # implent this method
-        pass  # delete this
+        self._name = name
+        self._phone_number = phone_number
 
     def is_phone_number_matching(self, input_phone_number):
-        # implent this method
-        pass  # delete this
+        return self._phone_number == input_phone_number
```

Comment:
@mmikiztli ha beveszed a normalizalast ide a `Person`-ba akkor nem is kell vele kivul foglalkoznod a `main.py`-ben.